gf-webapp/src/logic/contactsPage/ContactsPage.tsx

```diff
@@ -1,45 +1,29 @@
 import React, {useRef, useState} from "react";
 import {useDictionary} from "../DictionaryContext/useDictionary";
-import {send} from "emailjs-com";
 import {PageBorder} from "../../component/pageBorder/PageBorder";
+import {MailApiService} from "src/service/MailApi/MailApi";
+import {ApiAskAdminPostRequest} from "src/autogenerated/gf-server-api/models/ApiAskAdminPostRequest";
 import styles from "./ContactsPage.module.scss";
 import {Button} from "gf-ui-lib/components/Button/Button";
 
+const defaultAskAdminFormData: ApiAskAdminPostRequest = {name: "", email: "", phone: "", message: ""};
```

Comment:
bad format - set linter to put props on new lines

Reply:
done